hazelcast/src/test/java/com/hazelcast/executor/ExecutorServiceTestSupport.java

```diff
@@ -0,0 +1,197 @@
+package com.hazelcast.executor;
+
+import com.hazelcast.config.Config;
+import com.hazelcast.config.ExecutorConfig;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.HazelcastInstanceAware;
+import com.hazelcast.core.IExecutorService;
+import com.hazelcast.core.Member;
+import com.hazelcast.core.PartitionAware;
+import com.hazelcast.test.HazelcastTestSupport;
+
+import javax.script.ScriptEngine;
+import javax.script.ScriptEngineManager;
+import javax.script.ScriptException;
+import java.io.Serializable;
+import java.util.Map;
+import java.util.concurrent.Callable;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.fail;
+
+public class ExecutorServiceTestSupport extends HazelcastTestSupport {
+
+    IExecutorService createSingleNodeExecutorService(String name) {
+        return createSingleNodeExecutorService(name, ExecutorConfig.DEFAULT_POOL_SIZE);
+    }
+
+    IExecutorService createSingleNodeExecutorService(String name, int poolSize) {
+        final HazelcastInstance instance = createHazelcastInstance(
+                new Config().addExecutorConfig(new ExecutorConfig(name, poolSize)));
+        return instance.getExecutorService(name);
+    }
+
+    static class ScriptRunnable implements Runnable, Serializable, HazelcastInstanceAware {
```

Comment:
Perhaps an idea to move this inner class out of the ExecutorServiceTestSupport? The same goes for the  ScriptCallable. This makes the test easier to read since you don't need to deal with these details.